frontend/src/components/data-table/types.ts

```diff
@@ -38,3 +38,11 @@ export type DataTableSelection =
   | "single-cell"
   | "multi-cell"
   | null;
+
+export function shouldRenderDateInUTC(dtype: string | undefined): boolean {
+  if (!dtype) {
+    return false;
+  }
+  // Check for datetime[X,Y] format
+  return /^datetime\[[^,]+,[^,]+]$/.test(dtype);
```

Comment:
Should one of X or Y be "utc"?

Reply:
just posted a screenshot in in the description. whenever there is a timezone (not just UTC), polars at least renders as UTC.

Reply:
actually this might not be true. polars just converts it to UTC. let me see if I can get the column in polars in another tz

Reply:
yea i was wrong. i need to extract the timezone